engine/src/main/java/io/camunda/zeebe/engine/state/analyzers/CatchEventAnalyzer.java

```diff
@@ -201,15 +208,20 @@ private Optional<CatchEventTuple> findEscalationCatchEventInScope(
     final var elementType = processInstanceRecord.getBpmnElementType();
 
     final var element = process.getElementById(elementId, elementType, ExecutableActivity.class);
-    final Optional<ExecutableCatchEvent> catchEvent =
+    final Optional<ExecutableCatchEvent> escalstionCatchEvent =
```

Comment:
:wrench: There's a typo here:
```suggestion
    final Optional<ExecutableCatchEvent> escalationCatchEvent =
```